### PR TITLE
Find yards search form

### LINKED ISSUE #44: Search Index Page - Search again  Happy Path
As a user, when I visit search_index, and I properly fill out the location field, and purpose dropdown, then I can click search and do a new search

### LINKED ISSUE #45: Search Index Page - Search forms
As a user, when I visit search_index, I can see the page includes a new location field, and purpose dropdown that I can fill-out

### PR DESCRIPTION
**Pull request by Gvieve**
### Add
* partial for yards search form
* render partial on search index and find_yards views
* style search form
* logout test on landing_spec
* booking tests to get coverage to 100%

Closes #44 
closes #45

### CODE REVIEW

File: app/views/shared/_search_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    <%= form.label "Enter the zipcode where you want to yard:" %>
+    <%= form.text_field :location, placeholder: "Enter US Zipcode", required: true, class: "form-control-sm"%><br>
+    <%= form.label "How will you spend your time yarding?" %><br>
+    <%= form.collection_check_boxes(:purposes, purposes, :name, :name, include_hidden: false) do |purpose| %>
+      <%= purpose.check_box %>
+      <%= purpose.label %><br>
+    <% end %><br>
+    <%= form.submit 'Yard Me', class: "btn btn-secondary" %><br><br>
+  <% end %>
+</div>
```
Comment (amsmyth1) [inline app/views/shared/_search_form.html.erb:12]: PARTIAL QUEEN!

**Review by amsmyth1 — APPROVED** [review]
G, you are a G at partials :)

File: app/facades/booking_facade.rb [hunk 2]
```diff
@@ -1,24 +1,20 @@
...
+    return booking = {} if booking == {}
+    yard_info = yard_info(booking[:attributes][:yard_id])
+    OpenStruct.new({ id:           booking[:id],
+                     yard_id:      booking[:attributes][:yard_id],
+                     yard_name:    yard_info[:attributes][:name],
+                     status:       booking[:attributes][:status],
+                     name:         booking[:attributes][:booking_name],
+                     address:      full_address(yard_info),
+                     date:         booking[:attributes][:date].to_date,
+                     time:         get_time(booking),
```
Comment (domo2192) [inline app/facades/booking_facade.rb]: Tricked ya rspec!

**Review by domo2192 — APPROVED** [review]
Thank you!

**Pull request merged**